src/style.css

```diff
@@ -0,0 +1,189 @@
+/* #region Reset */
+:root {
+  box-sizing: border-box;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  --Main-accent: #060b35;
+  --Secondary: #616070;
+  --Blue-accent: #00acdc;
+
+}
+
+body {
+  margin: 0;
+}
+/* #endregion */
+/* #region Card */
+.card {
+  position: relative;
+  left: 0;
+  top: 0;
+  width: 198px;
+  height: 406px;
```

Comment:
1. position is unnecessary here
2. you can use display flex with flex-direction column 
3. width should have 200px with border, so just set it too 200px (check box-sizing once more)
4. it's better not to set height here, let's make content to dedicate it